cst438_registration/src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -0,0 +1,81 @@
+package com.cst438.controller;
+
+
+import com.cst438.domain.Enrollment;
+import com.cst438.domain.EnrollmentRepository;
+import com.cst438.domain.Student;
+import com.cst438.domain.StudentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class StudentController {
+    @Autowired
+    StudentRepository studentRepository;
+
+    @Autowired
+    EnrollmentRepository enrollmentRepository;
+
+    @GetMapping("/student/list")
+    public Iterable<Student> listStudents() {
+        return studentRepository.findAll();
+    }
+
+    @PostMapping("/student/add")
```

Comment:
This method is very comprehensive and readable. Comment is not too much, and not too vague. I am able to logically follow this method.